.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+name: OSX-build
+
+on: [push, pull_request]
+
+jobs:
+  osx-clang:
+
+    name: OSX build
+    runs-on: [macos-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: info
+      run: |
+        g++ -v
+        cmake --version
+    - name: configure
+      run: |
+        mkdir build
+        cd build
+        cmake ..
```

Comment:
AppleClang by default doesnt't support OpenMP, but with small modifications to the `CMake` invocations and installing the correct libraries, you can usually make it work:
https://gitlab.kitware.com/cmake/cmake/issues/18520

Do we have any influence over the build environment?

Reply:
Thanks, i will see if I can add this.

Reply:
A first step would be to see if Homebrew is installed and you can do `brew install libomp`